src/OmniSharp.Roslyn.CSharp/Services/Structure/CodeStructureService.cs

```diff
@@ -32,7 +32,8 @@ public class CodeStructureService : IRequestHandler<CodeStructureRequest, CodeSt
 
         public async Task<CodeStructureResponse> Handle(CodeStructureRequest request)
         {
-            var document = _workspace.GetDocument(request.FileName);
+            // To provide complete code structure for the document wait until all projects are loaded.
+            var document = await _workspace.GetDocumentFromFullProjectModelAsync(request.FileName);
```

Comment:
I noticed that Intellisense did not move onto the `FullProject` APIs. Because of that, you could open a file and start typing and get no intellisense.

Reply:
@rchande : the same is true currently as well - only after OnmiSharp loads all/some projects IntelliSense shows up. Or am I missing your question?